recipes/microhapulator/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.1.2" %}
+{% set sha256 = "d963fa011053204d02c0f37738333d7ca436577b60e358366cfc2a27dffbb244" %}
+
+package:
+  name: microhapulator
+  version: '{{version}}'
+
+source:
+  url: https://github.com/bioforensics/microhapulator/archive/{{ version }}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  noarch: python
+  entry_points:
+    - mhpl8r = microhapulator.__main__:main
+  script: python -m pip install --no-deps --ignore-installed .
+  number: 0
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - happer >=0.1
+    - insilicoseq >=1.3.5,!=1.3.5-py_0 
```

Comment:
Yeah, I tried something like this earlier and also failed the linter.

Reply:
It's not the linter... it just breaks in `conda`.

Reply:
Sorry, I forgot about this one.